repo-client/src/main/java/io/lionweb/repoclient/LionWebRepoClient.java

```diff
@@ -162,6 +168,27 @@ protected RepoClientConfiguration buildRepositoryConfiguration() {
     return jsonSerialization;
   }
 
+  //
+  // Raw Bulk APIs
+  //
+
+  public @Nullable RepositoryVersionToken rawCreatePartitions(@NotNull String data)
```

Comment:
Should the parameters really be strings? This forces to have them in memory. Maybe we can go with `CharSequence` -- that's implemented by `String`, so it's compatible, but at least leaves the option to use something more efficient.

Reply:
Or we directly go to `Stream`.

Reply:
I tried making it into a CharSequence but eventually we reach methods from okhttp that wants strings and cannot accept CharSequences